### DISCUSSION TITLE
5.x分支缺少转账到零钱功能

### DISCUSSION THREAD

**Discussion by sunkangchina** [w7corp/easywechat#2513] [Ideas]
对应官方文档 
https://pay.weixin.qq.com/wiki/doc/apiv3/apis/chapter4_3_1.shtml

**Comment by overtrue** [comment]
5.x 不支持  v3 哈， 6.x 起才支持 v3

**Reply by sunkangchina** [reply]
这就尴尬了。

**Reply by sunkangchina** [reply]
6.x PHP >= 8.0 , 其实不用一直升级版本，维护好一个分支，把功能完善就可以。
版本越升越高，PHP版本也变高，不就不好了。

**Reply by overtrue** [reply]
嗯，这是你的观点，没错。只是我不赞同。

**Reply by sunkangchina** [reply]
正常！

**Comment by TheNorthMemory** [comment]
支付单独包 [composer require wechatpay/wechatpay](https://packagist.org/packages/wechatpay/wechatpay) 你可以从PHP7.1一直用到PHP8.2，只要你喜欢，想咋用就咋用

**Reply by overtrue** [reply]
对，用它吧

**Reply by sunkangchina** [reply]
收到。感谢！
